src/code/z_message_nes.c

```diff
@@ -134,6 +95,20 @@ void Message_LoadPluralRupeesNES(PlayState* play, s16* decodedBufPos, s32* offse
     *arg3 = f;
 }
 
+char sRupeesTextLocalization[LANGUAGE_MAX - 1][8] = {
+    { "Rupee(s)" }, // EN
+    { "Rubin(e)" }, // DE
+    { "Rubis" },    // FR
+    { "Rupia(s)" }, // SPA
+};
+
+u8 sRupeesTextLength[LANGUAGE_MAX - 1] = {
+    sizeof("Rupee(s)") - 1,
+    sizeof("Rubin(e)") - 1,
+    sizeof("Rubis") - 1,
+    sizeof("Rupia(s)") - 1,
+};
```

Comment:
It may help to make the strings macros so they only need to be changed in one place.